### ISSUE TITLE
Crash on API 27

### ISSUE THREAD

**Issue by ANPez** [alhazmy13/MediaPicker#40]
The library crashes on API 27 (Android 8.1) because of the launched activity requesting orientation with a translucent theme, which is not allowed now.

```
Caused by: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
    at android.app.Activity.onCreate(Activity.java:987)
    at android.support.v4.app.SupportActivity.onCreate(SupportActivity.java:66)
    at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:297)
    at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:84)
    at net.alhazmy13.mediapicker.Video.VideoActivity.onCreate(VideoActivity.java:54)
```

**Comment by getglory** [comment]
I think he should remove "screenOrientation" in the AndroidManifest.

**Comment by alhazmy13** [comment]
Hi @ANPez 
Can you please confirm if it fixed wit last beta release [2.4.1-beta](https://github.com/alhazmy13/MediaPicker/releases/tag/2.4.1-beta) or not?

Thanks
